instance/openstack_utils.py

```diff
@@ -234,11 +235,17 @@ def download_swift_account(download_target, **kwargs):
                 if file_download_result['response_dict']['status'] == 304:
                     continue
 
-                errors[file_download_result['container']] += 1
+                errors[file_download_result['container']]['failures_count'] += 1
```

Comment:
nit: With the extra index here, the nested index that was already there is a bit confusing. Perhaps we could add a variable like so
```
container_name = file_download_result['container']
errors[container_name]['failures_count'] += 1
errors[container_name]['extra_information'].append(
...
```

Reply:
Good point!